.travis.yml

```diff
@@ -1,26 +1,33 @@
 language: python
+
 python:
-  - "3.5"
+- "3.5"
 
-# command to install dependencies
 install:
-  - pip install -r requirements.txt
+- pip install -r requirements.txt
 
 services:
-  - postgresql
+- postgresql
 
-# Create database for test
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
-  - python manage.py migrate --settings=core.settings
+- psql -c "create database travis_ci_test;" -U postgres
+- python manage.py migrate --settings=core.settings
 
-# command to run tests
 script:
-  - coverage run --source=app manage.py test --settings=core.settings
-  - flake8 .
+- coverage run --source=app manage.py test --settings=core.settings
+- flake8 .
 
-after_success:
-  coveralls
+after_success: coveralls
 
 notifications:
   slack: teamtaverna:iP55LweubU8UOBgu9e0cTeED
+
+deploy:
+  provider: heroku
+  api_key:
+    secure: CbUfdzEjhmzvedJGh/PUFZDsnJucro3ojwfaWjli/BZ+1s8WFv4JXGPhywJEdXyKABFm/qQeOVVDt8YMO5HSknUXaUuwA6F6XDEoQMr8J8sPprfU1wXiMzZ35IuNKHsXhV2eCep7JtU3gG6fA0KeOgjqwwZESd1y+HYWo792O3ilfUqH40aIjPTkYUFoXiweovhFycOK8P/O1z1wc3rT42zpPNef2i6y+/qCyQQD8zH7kibhKqUgTsILJtTogZ6Nuqa/bGQHWbTCsNl/udoNHknmBIfdEc2xiJnQKpsTJGOBphmqb2SnyO8Srop8qmYemm+6spDT1Uw4/8zPaKRhbwNXCAUvrII9jiUQXFbmfXfH3YmYjMhrbxU1Ieaujl8zYea6fTAaBmn3kdZLwhdFQQXosQrZW6y2pxxtrFYA52Df5tTpf5deFmJIAlwR9hI/cq4KY0ZfjnVRUOeTmvduKhmI+C6xZtlpYUfnCZ9msAuuvsxUJG7SisTilEKs6OULYuzHP9N5YlBfQ7EvO1FgsJi86/CBo/4/BEq5xDvMpOX55hwBnessT/Yc4nbu72yXO2iTb46SmS9ZhtttQx2QDv2E/yA6WCU7kLdtfprxT0n0q6haIyw7qNfAnEVHzHmASI9Vazu6BPSs1VsRRTGf1wSJc+masEdCa7RkHI4jatM=
```

Comment:
@andela-kerinoso That's the encrypted Heroku API key. Yes, it is safe pushing it to GitHub.